packages/assurance/README.md

```diff
@@ -1,31 +1,28 @@
 
-# React Native AEP Assurance Extension
+# React Native Adobe Experience Platform Assurance Extension
 
 [![npm version](https://img.shields.io/npm/v/@adobe/react-native-aepassurance/alpha?color=green&label=npm%20package)](https://www.npmjs.com/package/@adobe/react-native-aepassurance/v/3.0.0-alpha.1)
 [![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepassurance)](https://www.npmjs.com/package/@adobe/react-native-aepassurance/v/3.0.0-alpha.1)
 
 `@adobe/react-native-aepassurance` is a wrapper around the iOS and Android [AEPAssurance SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-assurance) to allow for integration with React Native applications. Functionality to start Assurance session is provided through JavaScript documented below.
 
-## Installation
+## Prerequisites
+
+The Adobe Experience Platform Assurance extension has the following peer dependency, which must be installed prior to installing the identity extension:
+- [Core](../core/README.md)
 
-You need to install the SDK with [npm](https://www.npmjs.com/) and configure the native Android/iOS project in your react native project. Before installing the Assurance extension it is recommended to begin by installing the [Core extension](../core/README.md).
 
-### Install npm package
+## Installation
 
-> Requires `@adobe/react-native-aepcore` to be installed.
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page
 
-Install the `@adobe/react-native-aepassurance` alpha package:
+Install the `@adobe/react-native-aepassurance` package:
 
 ```bash
 cd MyReactApp
-npm install @adobe/react-native-aepassurance@3.0.0-alpha.1
+npm install @adobe/react-native-aepassurance
 ```
 
-For iOS development, after installing the plugins from npm, download the pod dependencies by running the following command:
-`cd ios && pod install && cd ..`
-To update native dependencies to latest available versions, run the following command:
-`cd ios && pod update && cd ..`
-
 ### Link
 
```

Comment:
Do we need the Link Session here? Or we can point to the core Readme.
Can we do the similar way as other extension
Initialize the SDK via native code inside your AppDelegate and MainApplication in iOS and Android respectively, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).  So we make the connection for users to go to Core Readme.

We can still keep the Example of registering Assurance code here.